x-pack/plugins/apm/public/components/app/metrics/serverless_metrics/serverless_summary.tsx

```diff
@@ -163,6 +163,18 @@ export function ServerlessSummary({ serverlessId }: Props) {
           />
         </EuiFlexItem>
         {showVerticalRule && <VerticalRule />}
+        <EuiFlexItem grow={false}>
+          <EuiStat
+            isLoading={isLoading}
+            title={`$${data?.estimedCost}`}
+            titleSize="s"
+            description={i18n.translate(
+              'xpack.apm.serverlessMetrics.summary.estimatedCost',
+              { defaultMessage: 'Estimated costs avg.' }
```

Comment:
I think it would be useful to have a tooltip how the avg cost is calculated

Reply:
@boriskirov can you come up with a copy, please?